release_notes.md

```diff
@@ -37,3 +37,16 @@
 
 - Added support for OpenAI assistant
 
+- The AI Assistant can now understand and respond to context sensitive conversations and perform given SOC tasks.
+
+    - With bot's enhanced understanding of the natural language, it now derives its operating modes from the conversation; making the options &mdash; Playbook Blocks and Conversation Mode &mdash; redundant.
+
+    - The bot now has increased access to various modules, maintaining privacy is paramount. Sensitive data is still being actively masked for all interactions with the LLM.
+
+    - The bot can now hold conversations to understand requirements and generate a more targeted playbook.
```

Comment:
playbook generation experience has been streamlined with the introduction of the OpenAI assistant. Users can now engage with the bot during the creation of a playbook outline, making it easier to modify the outline by adding, updating, or removing steps.